R/bettr-main.R

```diff
@@ -543,7 +627,7 @@ bettr <- function(df, idCol = "Method",
                                 value = initialWeights[i],
                                 min = 0,
                                 max = 1,
-                                step = 0.05
+                                step = weightResolution
```

Comment:
As for the `initialWeightValue`, see comment above about promoting to main argument

Reply:
see comment above